ee/client/omnichannel/tags/TagsRoute.js

```diff
@@ -100,7 +100,7 @@ function TagsRoute() {
 
 	const header = useMemo(() => [
 		<GenericTable.HeaderCell key={'name'} direction={sort[1]} active={sort[0] === 'name'} onClick={onHeaderClick} sort='name' w='x120'>{t('Name')}</GenericTable.HeaderCell>,
-		<GenericTable.HeaderCell key={'description'} direction={sort[1]} active={sort[0] === 'description'} onClick={onHeaderClick} sort='description' w='x200'>{t('Description')}</GenericTable.HeaderCell>,
+		<GenericTable.HeaderCell key={'description'} direction={sort[1]} active={sort[0] === 'description'} onClick={onHeaderClick} sort='description' w='x60'>{t('Description')}</GenericTable.HeaderCell>,
```

Comment:
The remove button isn't aligned property anymore.

![Screen Shot 2020-11-18 at 10 18 19](https://user-images.githubusercontent.com/59577424/99535367-94e08d00-2987-11eb-9a40-49bd8f3428a2.png)

Reply:
I've reverted to "x200".